components/select/index.tsx

```diff
@@ -268,6 +274,8 @@ export default class Select<T = SelectValue> extends React.Component<SelectProps
         showArrow={showArrow}
         {...rest}
         {...modeConfig}
+        transitionName={disableAnimation ? null : 'slide-up'}
+        choiceTransitionName={disableAnimation ? null : 'zoom'}
```

Comment:
:ice_cream: 

May want to consider keeping `defaultProps` as is, extracting these from `this.props`, since it appears as though you're implicitly making the assumption that they won't be defined:

```
static defaultProps = {
    transitionName: 'slide-up',
    choiceTransitionName: 'zoom',
  };
```